toqito/state_metrics/fidelity_of_separability.py

```diff
@@ -16,8 +16,8 @@ def fidelity_of_separability(
     input_state_rho: np.ndarray,
     input_state_rho_dims: list[int],
     k: int = 1,
-    verbosity_option=2,
-    solver_option="cvxopt",
+    verbosity_option: int=2,
```

Comment:
The linter should catch this. Something I noticed was running pylint in `toqito/` and `toqito/tests` will catch the errors I worked on in this PR. 

But if I run pylint in a submodule, it will catch more issues: Like the line is too long etc. Trying to figure out a way to edit the pytest configuration file.